utils/zerovec/src/ule/error.rs

```diff
@@ -7,42 +7,48 @@ use core::fmt;
 
 /// A generic error type to be used for decoding slices of ULE types
 #[derive(Copy, Clone, Debug)]
-pub enum ULEError {
+pub enum ZeroVecError {
+    /// Attempted to parse a buffer into a slice of the given ULE type but its
+    /// length was not compatible
     InvalidLength { ty: &'static str, len: usize },
+    /// The byte sequence provided for `ty` failed to parse correctly
     ParseError { ty: &'static str },
-    FormatError,
+    /// The byte buffer was not in the appropriate format for VarZeroVec
+    VZVFormatError,
```

Comment:
Nit: I know it's more verbose but I would probably name this `VarZeroVecFormatError` since `VZV` is mostly an acronym we use among us developers but not one that we expect to be common jargon for users